docs/manual/amop_protocol.md

```diff
@@ -282,6 +264,106 @@ public class Channel2Client {
 }
 ```
 
+## Topic认证功能
+认证功能是指对于特定的topic消息，允许通过认证的接收者接收消息。
+
+2.1.0及之后的sdk和节点版本新增了topic认证功能，默认的配置没有开启认证功能，需要用到认证功能的话请参考[配置文件配置](./amop_protocol.html#id3)配置好公私钥，公私钥的生成方式请参考[生成公私钥脚本](./account.md)。使用过程如下：
+- 1：接收者使用[生成公私钥脚本](./account.md)生成公私钥文件，私钥保留，公钥给生产者。
+- 2：参考配置案例将配置文件配好。启动接收端和发送端进行收发消息。
+
+以逻辑架构图为例，说明认证的过程原理：
```

Comment:
It's better to add a framework graph here. Where is the public key and where is the private key.